xcode/Subconscious/Shared/Components/Common/MarkupTextViewRepresentable.swift

```diff
@@ -243,6 +239,52 @@ struct MarkupTextViewRepresentable: UIViewRepresentable {
                 )
             )
         }
+        
+        // MARK: - NSTextLayoutManagerDelegate
+                                
+        func textLayoutManager(
+            _ textLayoutManager: NSTextLayoutManager,
+            textLayoutFragmentFor location: NSTextLocation,
+            in textElement: NSTextElement
+        ) -> NSTextLayoutFragment {
+            let baseLayoutFragment = NSTextLayoutFragment(textElement: textElement, range: textElement.elementRange)
+            
+            guard let textContentStorage = textLayoutManager.textContentManager as? NSTextContentStorage else {
+                MarkupTextViewRepresentable.logger.warning("Could not access textContentStorage")
+                return baseLayoutFragment
+            }
+            
+            if let text = textContentStorage.attributedString(for: textElement)?.string {
+                let sub = Subtext(markup: text)
+                
+                // Only render transcludes for a single slug in a single block
+                if renderTranscludeBlocks && sub.slugs.count == 1 && sub.blocks.count == 1 {
```

Comment:
@gordonbrander I'm not sure if this is the usage of `Subtext` you were thinking of? Checking for a single slug and a single block isn't actually a sufficient heuristic for transcludes still, we need the link to be the only text in the block, correct?

Reply:
I'm realizing two things:

- The way we visualize Transcludes working is that wikilinks do not generate transcludes, while slashlinks DO generate transcludes.
    - Therefore, we want to be able to check for slashlink slugs, specifically. This feature does not currently exist in Subtext. We'll have to add it.
- It would be best to parse Subtext once, when block changes, and keep that instance around for reference in this delegate method, and others.
    - Currently MarkupViewRepresentable doesn't know about Subtext. Instead we have a `renderAttributesOf` delegate, which takes a mutable attributed string and applies attributes. The intention was to make it generic over many markups.
    - However, it may simplify things to complect MarkupTextViewRepresentable with Subtext and call it SubtextTextViewRepresentable instead. This would allow us to make Subtext domain logic available in more places in the implementation. E.g. keep a parsed Subtext instance on the custom textview instance.

All that said, I think we should remove this logic for now, land this PR, then sketch out how we want to approach bringing in transcludes in a follow-up PR.

Reply:
As discussed on call, this looks like the right approach going forward. I will remove the transclude detection from this PR altogether and we'll land it via #424 instead.

Reply:
I've removed all the transclude related code from this PR.